src/lib/serviceRoutinesV2/optionsAllNotDelete.cpp

```diff
@@ -45,8 +47,11 @@ std::string optionsAllNotDelete
   ParseData*                 parseDataP
 )
 {
-  ciP->httpHeader.push_back("Access-Control-Allow-Methods");
-  ciP->httpHeaderValue.push_back("GET, PATCH, POST, PUT, OPTIONS");
+  if ( isOriginAllowedForCORS(ciP->httpHeaders.origin) )

```

Comment:
I think is should be

`if (isOriginAllowedForCORS(ciP->httpHeaders.origin))`

per [M11 style rule](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#m11-spaces)

> Also, after `(` and before `)`, spaces MUST NOT be present.

Reply:
(Seen in other part of the PR also)

Reply:
Fixed in da358b6